make/test.py

```
@@ -62,29 +63,30 @@ def command(args: Namespace) -> None:
+            "create",
+            "-f",
+            "qcow2",
+            "/workspace/disk.qcow2",
+            "32G",
         )
         kernel, initrd, uuid = extract_boot(iso, workspace, branch)
         disk: str = "/workspace/disk.qcow2"
 
         # Phase 1: boot the iso, validate, install to the target disk.
+        cidfile: str = os.path.join(workspace, "phase1.cid")
         proc: subprocess.Popen[bytes] = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)